Tools/LyTestTools/ly_test_tools/o3de/asset_processor.py

```diff
@@ -209,7 +209,7 @@ def _get_port_from_log():
                     logger.debug(f"Read port type {port_type} : {port}")
                     return True
                 except Exception as ex:  # intentionally broad
-                    logger.debug("Failed to read port from file", exc_info=ex)
+                    logger.warning(f"Failed to read port type {port_type} : {port} from file", exc_info=ex)
```

Comment:
I seem to remember this message was set to debug to reduce spam. Line 220 can result in many read attempts before the expected line get written to this file.

Reply:
set back to debug